src/fundus/publishers/fr/le_figaro.py

```diff
@@ -0,0 +1,37 @@
+import datetime
+from typing import List, Optional
+
+import lxml
+from lxml.cssselect import CSSSelector
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+)
+
+
+class LeFigaroParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector: CSSSelector = CSSSelector("p[class='fig-paragraph']")
+        _author_selector: CSSSelector = CSSSelector("a[rel='author']")
```

Comment:
`_author_selector` looks like a leftover.

Further, the publisher uses a little summary to introduce their articles as well as subheadlines between sections of paragraphs. Could you add those? Here is an [example article](https://etudiant.lefigaro.fr/article/etudes/a-strasbourg-rennes-menton-saint-etienne-des-etudiants-bloquent-les-iep-20240430/) where you can find both.

After adding, you have to run `python -m scripts.generate_parser_test_files -p LeFigaro -oj`

Reply:
I will add that!
What do you mean by "leftover"? This is how I was able to extract the authors name..